app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
I'd move this helper into a helper specific to the resource using it. e.g, `ArticleHelper`